.github/workflows/nroff-elves.sh

```diff
@@ -51,7 +51,7 @@ done
 git config --global user.name "OFIWG Bot"
 git config --global user.email "ofiwg@lists.openfabrics.org"
 
-branch_name=pr/update-nroff-generated-man-pages-$BASE_REF
+branch_name=pr/update-nroff-generated-man-pages-$BASE_REF-`date +%s`
```

Comment:
Does the %s just pick up something random?

Reply:
the command would print "seconds since the Epoch".